pymatgen/io/lobster/outputs.py

```diff
@@ -408,6 +409,109 @@ def icohpcollection(self):
         return self._icohpcollection
 
 
+class NciCobiList:
+    """
+    Class to read NcICOBILIST (multi-center ICOBI) files generated by LOBSTER.
+
+    Attributes:
+        is_spin_polarized (bool): Boolean to indicate if the calculation is spin polarized.
+        NciCobiList (dict): Dict containing the listfile data of the form:
+        {bond: "number_of_atoms": number of atoms involved in the multi-center interaction,
+                "ncicobi": {Spin.up: Nc-ICOBI(Ef) spin up, Spin.down: ...}},
+                "interaction_type": type of the multi-center interaction
+    """
+
+    def __init__(self, filename: str | None = None):
+        """
+        Args:
+            filename: Name of the NcICOBILIST file.
+        """
+
+        if filename is None:
+            warnings.warn("Please consider using the newest LOBSTER version (4.1.0+). See http://www.cohp.de/.")
```

Comment:
Is this warning correctly indented? Why only warn is `filename is None`?

Reply:
It looks like a leftover from the icohplist. I remember implementing this (with another if clause). I guess this can be removed.

Reply:
Wouldn't you still use the default naming option in such a case and then then the file wouldn't be there? Thus, shouldn't it be the warning if the default file name does not exist rather than when it is set to `None`?